tests/site/xss_safe.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>XSS Safe</title>
+</head>
+<body>
+    <script>
+        window.escaped = "\u0027\u0027\u003B!\u002D\u002D\u0022\u003CXSS\u003E\u003D\u0026{()}";
+    </script>
+    <div id="fixture">
+        <div class="escaped">&#39;&#39;;!--&#34;&lt;XSS&gt;=&amp;{()}</div>
+        <input type="text" value="&#39;&#39;;!--&#34;&lt;XSS&gt;=&amp;{()}">
```

Comment:
@cahrens - @efischer19 determined that it wasn't that this page source shouldn't be escaped just as everything else is escaped.  The problem is that the `innerHtml` call off the document element outputs values with a different set of rules, so the regex needs to account for that.

Note that when we stopped the test on this page, and used View Page Source from the browser, we would again see this original fully escaped text.